source/github-events/on-new-review.ts

```diff
@@ -0,0 +1,13 @@
+import delegate from 'delegate-it';
+
+function callbackWrapper(callback: VoidFunction): (event: CustomEvent) => void {
+	return function (event) {
+		if (/^pull request #\d+ updated$/.test(event.detail.data.reason)) {
```

Comment:
Not sure if the test is really needed, it seems all the messages are the same (with just the id changing).

Reply:
The issue is that this event might also be dispatched when a regular comment comes in, so it would effectively duplicate `onNewComment` as well.

Reply:
> The issue is that this event might also be dispatched when a regular comment comes in

It is, but not from the same source, so the delegation ignores it.

The issue is that it is dispatched three times when sending a quick review (with "Add single review comment" button), and afaict the three events are virtually indistinguishable :grimacing:

Reply:
That doesn't sound right. What's happening though is that we have a simple loop with deduplication in `init`. So even if it's called twice, no duplication happens.

It may be ok, but I'm still wary of this event and I'd probably merge it back into this file so we know it's not safe for general use. A comment like this should also be inlined.